include/pagmo/algorithms/mbh.hpp

```diff
@@ -393,6 +393,16 @@ class mbh
     {
         return m_algorithm;
     }
+    /// Setter for the inner algorithm.
```

Comment:
forgot to remove it .... I had added it as I wanted to replicate nlopt API for the local_optimizer, where the property is exposed in python as not read-only.

Then I changed my mind but forgot to remove this.